src/legacy/ui/public/vis/editors/default/_agg_params.scss

```
@@ -10,3 +10,12 @@
   color: $textColor;
   background-color: $backgroundColor;
 }
+
+.visEditorAggParam--inline {
```

I'm future thinking on this and it might be better to explicitly state that this will be **half** the size and inline. So can you change the class to
```suggestion
.visEditorAggParam--half {
```

---

Make sense. Done!